api.js

```diff
@@ -1,9 +1,17 @@
 var mongo = require('mongodb').MongoClient,
-    fiddles = null;
+    fiddles = null,
+    mongoConnectionStr = process.env.MONGOHQ_URL ?
```

Comment:
But there should be something to fallback to right ? and yup ES6FIDDLE_DB_URL makes sense..